windows-driver-docs-pr/devtest/28720-banned-api-usage-use-updated-function-replacement.md

```diff
@@ -9,9 +9,23 @@ f1_keywords:
 # C28720
 
 
-warning C28720: Banned API Usage
+**Warning C28720: IsBadXXXPtr API Usage**\
+Example output: ```Banned API Usage:  *function name* is a Banned API as listed in dontuse.h for security purposes.```
```

Comment:
This example output is affected by my recent change:
```Banned API Usage:  *function name* is insecure and has been marked deprecated.```

Reply:
Changed to reflect